### PR TITLE
Temporary fix - add cache buster to maps thumbnails

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/DAT-2491

/cc @RafLeszczynski 

You can check it working here http://mediawiki119.igor.wikia-dev.com/wiki/Special:Maps

### CODE REVIEW

**Comment by RafLeszczynski** [comment]
LGTM

File: extensions/wikia/WikiaMaps/controllers/WikiaMapsSpecialController.class.php [hunk 1]
```diff
@@ -319,8 +319,16 @@ public function translation() {
 	 * @param Integer $height
 	 */
 	private function convertImagesToThumbs( &$items, $width, $height ) {
+		/*
+		 FIXME this is a temporary fix for MWEB-1100
+		 We have 404s cached for a week and need to add cache buster until they are expired
+		 This should be removed on 2015-02-16 (first Monday after expiration time)
+		 */
+		$cacheBuster = '?cb20150206';
```
Comment (rogatty) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsSpecialController.class.php]: How about `?cb=20150206`? It probably doesn't make a difference but looks better.

File: extensions/wikia/WikiaMaps/controllers/WikiaMapsSpecialController.class.php [hunk 2]
```diff
@@ -319,8 +319,16 @@ public function translation() {
...
+
 		foreach ( $items as $item ) {
-			$item->image = $this->getModel()->createCroppedThumb( $item->image, $width, $height );
+			$thumb = $this->getModel()->createCroppedThumb( $item->image, $width, $height );
+			$item->image = $thumb . $cacheBuster ;
```
Comment (rogatty) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsSpecialController.class.php]: There should be no space before semicolon.

**Comment by rogatty** [comment]
LGTM

**Pull request merged**